source/daplink/RTX_Config.c

```diff
@@ -90,7 +90,7 @@
 #elif defined(INTERFACE_SAM3U2C)
 #define OS_CLOCK    96000000
 #elif defined(INTERFACE_LPC4322)
-#define OS_CLOCK    204000000
+#define OS_CLOCK    96000000
```

Comment:
I checked with the LPC team internally, their recommendation was to slow the core clock down as the chip is not stable when running at this speed.